src/view/dataElements/identityMap/fetchNamespaces.js

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import getBaseRequestHeaders from "../../utils/getBaseRequestHeaders";
+import platform from "../xdmObject/helpers/platform";
+
+export default ({ orgId, imsAccess }) => {
+  const headers = getBaseRequestHeaders({ orgId, imsAccess });
+
+  return fetch(`${platform.getHost()}/data/core/idnamespace/identities`, {
```

Comment:
Hey @ninaceban, we will want to include the change that I just made in #108 to this

Reply:
thank you! I will update with latest